File: 14-Dictionaries/29-Dictionary-Access.js
const sample = require('lodash/sample')

let food = sample(["cheese pizza", "quiche","morning bun","gummy bear","tea cake"])

let bakeryStock = {
    "almond croissant" : 12,
    "toffee cookie": 3,
    "morning bun": 1,
    "chocolate chunk cookie": 9,
    "tea cake": 25
}

// first way
if (bakeryStock[food]) {
    console.log(`${bakeryStock[food]} left`)
} else {
    console.log("We don't make that")
}


// second way
let quantity = bakeryStock[food]
if (quantity) {
    console.log(`${quantity} left`)
} else {
    console.log("We don't make that")
}

File: 14-Dictionaries/29-Dictionary-Access.py
from random import choice
food = choice(["cheese pizza", "quiche","morning bun","gummy bear","tea cake"]) #DON'T CHANGE!

bakery_stock = {
    "almond croissant" : 12,
    "toffee cookie": 3,
    "morning bun": 1,
    "chocolate chunk cookie": 9,
    "tea cake": 25
}

# first way
if food in bakery_stock:
    print("{} left".format(bakery_stock[food]))
else:
    print("We don't make that")

# second way
quantity = bakery_stock.get(food)
if quantity:
    print("{} left".format(quantity))
else:
    print("we don't make that")